### PR TITLE
 Display code in search engine instead of redirecting

### PR DESCRIPTION
**Pull request by maddaladivya**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [x] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: 
fixes: #37 

This works only when DEBUG = True

### CODE REVIEW

**Comment by maddaladivya** [comment]
I tried doing it without scrapping.

File: search/views.py [hunk 1]
```diff
@@ -87,3 +86,9 @@ def subsq(a, b, m, n):
         return subsq(a, b, m - 1, n - 1)
     # If last characters are not matching
     return subsq(a, b, m, n - 1)
+
+def display(request):
+    if request.method == 'POST':
+        display = request.POST.get('path')
+    r = requests.get(display)
+    return render(request, 'cosmos/data.html',{'code':r.text})
```
Comment (AdiChat) [inline search/views.py]: It looks like you have not included `data.html` in this pull request. Kindly include it.

**Comment by Pl4gue** [comment]
@AdiChat We may consider squashing this PR before merging :+1:

**Comment by AdiChat** [comment]
@maddaladivya Instead of using POST requests, you may use query parameters as it is important to have shareable links. 
If we have shareable links, further socializing features can be developed over this. 

You may take a look at this small change. :+1:

**Comment by AdiChat** [comment]
Great work :tada: 

The feature looks good and a few minor changes will make it perfect. 
* On the results page, the code page should open in a new tab. 

* The url format is like `https://search.opengenus.org/display/?csrfmiddlewaretoken=nMsTfxHZ7sxGdrwAmNGM7Ku0cwWAYZ9r3btexorfQf8hD8BrDr6Qo6wGn343yK3B&path=https%3A%2F%2Fraw.githubusercontent.com%2FOpenGenus%2Fcosmos%2Fmaster%2Fcode%2Fbacktracking%2Fsrc%2Fsubset_sum%2FSubsetSum.java`. It would ne nice if the url is of the format like `https://search.opengenus.org/display?path="backtracking/src/subset_sum/SubsetSum.java"`. This will make the url sharable friendly and more natural. 

Kindly take a look at the above issues and make appropriate changes. :+1:

**Comment by maddaladivya** [comment]
Yes, I am working on it. Almost done and I just have to add a last feature to it.

**Comment by maddaladivya** [comment]
@AdiChat please review.

**Comment by AdiChat** [comment]
Great work :+1:

You may place a light-weight header. For the path `graph_algorithms/src/graph_coloring/graph_coloring.cpp`, the header may be `graph algorithms / graph coloring / graph coloring in C++`.

Additionally, you may place a small button on the top like "Ask any questions" linking to our discourse forum. This is for the time being until we integrate it with the searches. 

For the code area, keep a small left padding and include line numbers. Additionally, include a `copy to clipboard` option. 

Kindly take a look into the above minor modifications and make appropriate changes. :+1:

**Comment by AdiChat** [comment]
Perfect :+1:

The "Ask any question" button must open [OpenGenus Discourse](https://discourse.opengenus.org/) in a new tab and on clicking on the "copy to clipboard" button, a lightweight notification must be displayed that the code has been copied. 

Kindly make these minor changes and it will be good to be merged. :+1:

**Comment by maddaladivya** [comment]
@AdiChat  please review.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**